src/App.js

```diff
@@ -1,11 +1,25 @@
-import React, { useState, useEffect } from 'react';
+import PubSub from 'pubsub-js';
+import React, { useEffect, useState } from 'react';
 import './App.css';
 
+const MFE_TOPIC = 'MFE Topic';
```

Comment:
This is the topic where the messages are published.
The app also listens to the topic.